<!DOCTYPE html>
<html lang="en"><head>
    <meta http-equiv="Content-type" content="text/html; charset=UTF-8">
    <title>Mathematical Medicine LLC</title>
    <style type="text/css" media="all">
        body {
            background-color: #eee;
            color: black;
            box-sizing: border-box;
            max-width: 50rem;
            margin: 0 auto;
            font-family: verdana, arial, helvetica, sans-serif;
            padding-top: 1vh;
            padding-bottom: 1vh;
            min-height: 100vh;
            overflow-y: scroll;
            /*text-align: justify;*/
        }
        h1 { text-align: center; }
        h1 img { width: 100%; }
        span.sc { font-variant: small-caps; }
        p#copyright {
            margin: 0.2rem;
            text-align: center;
            font-size: 0.6rem;
        }
        /* Tab system adapted from https://codepen.io/mikestreety/pen/yVNNNm */
        /* License for same available on that page. Thanks, mikestreety! */
        .tabs {
            display: flex;
            flex-wrap: wrap;
        }
        .tabs label {
            order: 1;
            display: block;
            padding: 0.5rem 1rem;
            margin-right: 0.2rem;
            cursor: pointer;
            background-color: lightblue;
            font-weight: bold;
            transition: background-color ease 0.2s;
            box-sizing: border-box;
            width: 12.35rem; /* 50rem, 4 tabs, 0.6rem in margins */
        }
        .tabs label.lasttab { margin-right: 0; }
        .tabs label.faketab { display: none !important; }
        .tabs .tab {
            order: 99;
            flex-grow: 1;
            width: 100%;
            display: none;
            padding: 1rem;
            background-color: #fff;
        }
        .tabs input[type="radio"] {
            display: none;
        }
        .tabs input[type="radio"]:checked + label {
            background: #fff;
        }
        .tabs input[type="radio"]:checked + label + .tab {
            display: block;
        }
        @media (max-width: 45em) {
            .tabs .tab,
            .tabs label {
                order: initial;
            }
            .tabs label {
                width: 100%;
                margin-right: 0;
                margin-top: 0.2rem;
            }
        }
    </style>
    <script type="text/javascript">
        /* just some really simple history/bookmarkability management */
        function properPage() {
            let tab = document.getElementById(location.hash.replace('#', ''));
            tab = tab ? tab : document.getElementById('home');
            tab.checked = true;
        }
        window.addEventListener('click', event => {
            if (event.target.name == "tabs") {
                window.location.hash = event.target.id;
            }}
        );
        window.addEventListener('hashchange', properPage);
        window.onload = properPage;
    </script>
    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
    <link rel="manifest" href="/site.webmanifest">
</head><body>
    <h1><a href="#home"><img src="LogoAndWordmark-Banner.png"></a></h1>
    <div class="tabs">
        <input type="radio" name="tabs" id="home">
        <label for="home">Home</label>
        <div class="tab">
            <p>Mathematical Medicine LLC is a research organization specializing in statistical methodology and large-scale data curation projects, with a particular emphasis on statistical genetics and genetic data.</p>
            
            <p>For any inquiries, <a href="mailto:contactus@mathmed.org">contactus@mathmed.org</a>.</p>
        </div>
        
        <input type="radio" name="tabs" id="whoweare">
        <label for="whoweare">Who We Are</label>
        <div class="tab">
            
            <h3>Veronica Vieland, PhD</h3>
            <p><em>Executive Director</em></p>
            <p>Dr. Veronica Vieland is the founder and Executive Director of Mathematical Medicine LLC. Throughout her academic career, Dr. Vieland’s research has focused on the measurement of statistical evidence, both from a theoretical perspective and also as the foundation for understanding human genetic and genomic data. On the genetics side, she focuses on developing statistical methods for discovering the genetic architecture of complex clinical phenotypes. She has led development of a novel class of statistical genetic techniques, implemented in the computer package <span class="sc">Kelvin</span>, and used <span class="sc">Kelvin</span> to study the genetics of numerous clinical conditions, including autism, schizophrenia, bipolar disorder, cleft lip and palate, autoimmune thyroid disease, papillary thyroid cancer and genetic modifiers in Duchenne muscular dystrophy. The other arm of this work has been the development of the high-performance computational approaches required to utilize the methods. On the theoretical side, she is developing an absolute scale for the measurement of statistical evidence, following the template of the absolute scale for measurement of temperature.</p>
            <p>Dr. Vieland received her undergraduate degree in Philosophy from Barnard College and graduate degrees in Philosophy (Mathematical Logic and Philosophy of Science) and Biostatistics from Columbia University, where she also completed a Postdoctoral Fellowship in the Department of Child Psychiatry. Subsequent to completing her training, she was an academic for over 30 years, most recently serving as the Battelle Endowed Chair in Computational Biology at The Abigail Wexner Research Institute at Nationwide Children’s Hospital, where she also served as Vice President for Computational Research and as the Director of the Battelle Center for Mathematical Medicine, and as Professor in the Departments of Pediatrics and Statistics at The Ohio State University, where she remains Professor Emerita. She has published over 150 papers and presented her work at numerous national and international venues. She has been the recipient of two NIH Career Development Awards, and she is an elected fellow of the American Psychopathological Association (APPA) and the American Association for the Advancement of Science (AAAS).</p>
            <p>An <a href="#contributions">overview of Dr. Vieland’s contributions to science</a> and <a href="Vieland%20CV%20May%202022.pdf">her full CV</a> are available.</p>
            
            <h3>William Valentine-Cooper</h3>
            <p><em>Senior Systems Developer</em></p>
            <p>William has a bachelor’s degree in Computer and Information Science from The Ohio State University, and has worked for over 40 years as a systems architect, developer and project leader in many different industries, especially telecommunications. He has extensive experience with a wide variety of operating systems, languages, databases and tools, and has developed software ranging from high-performance device drivers to accounting systems and cellular network alarm management systems.  He has worked with Dr. Vieland since 2007. He is developing software and databases for MathMed research projects, managing application performance, and providing general technical support as required.</p>
            
            <h3>Jo Valentine-Cooper</h3>
            <p><em>Applications Developer</em></p>
            <p>Jo is a “second-generation computer scientist”, formally educated at Bowling Green State University. She has over 25 years of experience with systems administration, website programming, applications development, and user technical support using many different computing environments, and has worked with Dr. Vieland since early 2013. At MathMed, she’s employed in applications development, testing, documentation, quality control, and IT support.</p>
            <!--she's also to blame for the website-->
            
            <h2>Consultants</h2>
            
            <h3>John Burian, MS</h3>
            <p><em>Senior Systems Developer</em></p>
            <p>John received his MS in Computer Science from Bowling Green State University in 1992, and has worked with Dr. Vieland since 2007. In his work for MathMed, John develops parallel computation systems for genetic analysis, implements pipelines for data analyses on clinical research projects, and provides supplemental system administration. He also continues to work with Dr. Vieland on various university-based research grants.</p>
            
            
            <h3>Sang-Cheol Seok, PhD</h3>
            <p><em>Systems Programmer</em></p>
            <p>Sang-Cheol received his BS and MS degrees in Mathematics at the Pusan National University, Pusan, Korea and a PhD in Computer Science at the University of Iowa. He has expertise in Data mining, High Performance Computing, Numerical Analysis and Combinatorial Scientific Computing. He has worked with Dr. Vieland since 2007, acquiring specialized expertise in the underlying likelihood calculations used by <span class="sc">Kelvin</span>, and developing <span class="sc">Kelvin</span>’s unique multidimensional numerical integration algorithm. In his work for MathMed, he works with Dr. Vieland on the development and application of evidence measurement scales for biomedical research. He also continues to work with Dr. Vieland on university-based research grants.</p>
            
            <h3>Susan Valentine-Cooper</h3>
            <p><em>Business Manager</em></p>
            <p>Susan has a BS in MIS and Accounting from The Ohio State University and 40 years of income tax preparation experience. While a programmer/tax specialist with CompuServe, she helped develop HR Block’s original conversion to computer-based tax prep. She has extensive experience with accounting and tax compliance for small businesses, estates, and non-profits. Having been a Senior Project Manager, Executive Director for a synagogue, and entrepreneur, she has wide knowledge of budgeting, bookkeeping, business workflows, and government compliance, which she applies as the business manager for Mathematical Medicine LLC.</p>
        </div>
            
        <input type="radio" name="tabs" id="contributions">
        <label for="contributions" class="faketab"></label>
        <div class="tab">
            <h3>Contributions to Science</h3>
            
            <h4>Measurement of Statistical Evidence</h4>
            <p>Assessment of the strength of evidence for or against hypotheses is a key endpoint of statistical analysis throughout the biomedical sciences, and a variety of statistics are used as evidence measures (p-values, likelihood ratios, inter alia). While there are lively ongoing debates regarding whether p-values or various alternatives ought to be treated as evidence measures, one issue has received virtually no attention, namely, the question of measurement calibration. Our standard measures of evidence lack calibration with the result that a given change in any particular measure does not always correspond to the same amount of change in the strength of evidence; different measures are on fundamentally different scales; and any one of them may even indicate decreasing strength of evidence while the evidence strength is actually increasing. Just as failure to properly calibrate experimental equipment can lead to scientific errors, uncalibrated evidence measures can lead to erroneous statistical interpretations of biological data. The goal of this project is to develop an absolute (context-independent) measure of the strength of evidence, through a novel information-dynamic paradigm, in which distinct forms of information are related to one another as conserved and inter-convertible quantities, and units of evidence are defined in terms of these relationships. Work on this project has been supported by the Keck Foundation.</p>
            
            <p>Selected References:</p>
            <ul class="papers">
                <li>Vieland VJ. <a href="https://doi.org/10.1159/000093775">Thermometers: Something for statistical geneticists to think about.</a> Human Hered, 61:144-156, 2006. PMID: 16770079, doi:10.1159/000093775</li>
                <li>Vieland VJ, Das J, Hodge SE, Seok S-C. <a href="https://doi.org/10.1007/s12064-013-0180-9">Measurement of statistical evidence on an absolute scale following thermodynamic principles.</a> Theory in Biosciences 132(3):181-194, 2013. PMID: 23463577, PMC3742421, doi:10.1007/s12064-013-0180-9</li>
                <li>Vieland VJ, Seok S-C. <a href="https://doi.org/10.3390/e17085333">Statistical evidence measured on a properly calibrated scale across nested and non-nested hypothesis comparison.</a> Entropy 17:5333-5352, 2015. doi:10.3390/e17085333</li>
                <li>Vieland VJ, Seok S-C. <a href="https://doi.org/10.3390/e18040114">Statistical evidence measured on a properly calibrated scale for multinomial hypothesis comparisons.</a> Entropy 18(4):114, 2016. doi:10.3390/e18040114</li>
                <li>Vieland VJ. <a href="https://doi.org/10.1086/693931">Measurement of Statistical Evidence: Picking up Where Hacking (et al.) Left Off.</a> Philosophy of Science 84(5):853-865, 2017. doi:10.1086/693931</li>
            </ul>
            
            <h4>Statistical Genetic Theory</h4>
            <p>We focus here on one particular topic. Family-based human genetic studies usually employ opportunistic sampling of probands (individuals with the phenotype of interest), together with intrafamilial sampling to bring in relatives. Non-random proband selection together with genetically structured non-independence among relatives within families introduces a statistical complication that is unique to statistical genetics, giving rise to what is known as the problem of ascertainment. Early in Dr. Vieland’s career, ascertainment posed a very practical problem for the gene mapping community. It was widely believed that LOD scores depended on specification of the correct trait model, which could only be estimated from a prior segregation analysis using rigorous ascertainment protocols; but these studies were difficult to conduct, and for complex disorders they generally failed to yield definitive results. How then were we to choose a (presumably wrong) model to use for linkage analysis? And how could the results of analysis be interpreted given that modeling assumptions were almost certainly wrong? While this problem led many to move to so-called “model-free” methods, Dr. Vieland took a different tack. In a series of papers, together with SE Hodge, she laid out a general likelihood framework in which all existing ascertainment models could be represented as special cases. Using this framework we were able to prove a number of new and surprising results. Most importantly, this led to a new understanding of the true role of ascertainment in linkage analysis, opening the door to the use of far more powerful maximum likelihood or Bayesian methods for handling the unknown trait model. Our 1995 paper is one of an elite group cited for their fundamental contributions in the review paper “A Century of Biometrical Genetics” (Biometrics 56, 2000), written by two of the field’s luminaries, Robert Elston and Elizabeth Thompson. In additional theoretical papers, Dr. Vieland derived fundamental results related to robust approximations for modeling oligogenic traits, age-of-onset anticipation, modeling of gene x gene interactions; and Bayesian sequential updating as an effective alternative to independent replication.</p>
            
            <p>Selected References:</p>
            <ul class="papers">
                <li>Vieland VJ, and Hodge SE. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1801326/">Inherent intractability of the problems of ascertainment for pedigree data: A general likelihood approach.</a> Am J Hum Genet, 56:33-43, 1995. PMID: 7825595, PMC1801326</li>
                <li>Vieland VJ, Hodge SE. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1914614/">The problem of ascertainment for linkage analysis.</a> Am J Hum Genet 58(5):1072-1084, 1996. PMID: 8651268, PMC1914614</li>
                <li>Hodge SE, Vieland VJ. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1207613/">The essence of single ascertainment.</a> Genetics 144(3):1215-1223, 1996. PMID: 8913762, PMC1207613</li>
                <li>Vieland VJ, Logue M. <a href="https://doi.org/10.1159/000048601">HLODs, trait models, and ascertainment: Implications of admixture for parameter estimation and linkage detection.</a> Hum Hered 53:23-35, 2002. PMID: 11901268, No PMC#, doi:10.1159/000048601</li>
            </ul>
            
            <h4>Statistical &amp; Computational Methods Development</h4>
            <p>Around 1998, Dr. Vieland began to question the idea that Bayesian methods, which required specification of prior distributions, were therefore “subjective” and not appropriate for genetic research. But in application to family data, Bayesian methods are also computationally far more complex than applications in other statistical areas (again, for reasons related to the structure of correlations among individuals within families, as well as issues of ascertainment). Moreover exact methods for integrating over modeling parameters (as Bayesian analysis does) were computationally out of reach for all but the most trivial trait models, while existing MCMC samplers were not effective in dealing with unknown trait parameters. She therefore forged collaborations with computer scientists and software engineers in order to expand the scope of what could be computed. This led to the development of the robust and powerful techniques implemented in our software package <span class="sc">Kelvin</span>.</p>
            
            <p>Selected References:</p>
            <ul class="papers">
                <li>Vieland VJ. <a href="https://doi.org/10.1086/302076">Bayesian linkage analysis, or: How I learned to stop worrying and love the posterior probability of linkage.</a> Am J Hum Genet 63(4):947-954, 1998. PMID: 9758634, PMC1377508, doi:10.1086/302076</li>
                <li>Seok S-C, Evans M, Vieland VJ. <a href="https://doi.org/10.1089/cmb.2008.0175">Fast and accurate calculation of a computationally intensive statistic for mapping disease genes.</a> J Comput Biol, 16(5):659-676, 2009. PMID: 19432537, PMC3148122, doi:10.1089/cmb.2008.0175</li>
                <li>Vieland VJ, Huang Y, Seok S-C, Burian J, Catalyurek U, O’Connell J, Segre A, Valentine-Cooper W. <a href="https://doi.org/10.1159/000330634">KELVIN: a software package for rigorous measurement of statistical evidence in human genetics.</a> Hum Hered 72(4):276-288, 2011. PMID: 22189470, PMC3267994, doi:10.1159/000330634</li>
                <li>Huang Y, Thomas A, Vieland VJ. <a href="https://doi.org/10.3389/fgene.2013.00059">Employing MCMC under the PPL framework to analyze sequence data in large pedigrees.</a> Frontiers in Applied Genetic Epidemiology 4:59, 2013. PMID: 23626600, PMC363090, doi:10.3389/fgene.2013.00059</li>
            </ul>
            
            <h4>Clinical Applications of Statistical Methods</h4>
            <p>Throughout Dr. Vieland’s career she has collaborated actively on clinical genetic studies, applying a variety of techniques, including state-of-the-art data analytic methods developed by others, as well as statistical techniques developed in-house by her group. Here we cite a few of the publications that exemplify this work: (1) uses epistasis analyses to model the role of an associated allele of CTLA4 in modulating the effect of multiple different major genes on AITD, demonstrating that the same allele can be either protective or deleterious depending on genetic background; (2) and (3) represent contributions of our statistical methodologies to psychiatric genetics, utilizing combined linkage, LD-based fine-mapping and genome-wide association approaches; and (4) applies these same methods to a set of large families phenotyped for complex quantitative traits involved in musical aptitude, finding supporting evidence for both ear-development and neurocognitive genes.</p>
            
            <p>Selected References:</p>
            <ul class="papers">
                <li>Vieland VJ, Huang Y, Bartlett CW, Davies TF, Tomer Y. <a href="https://doi.org/10.1016/j.ajhg.2008.04.017">A multilocus model of the genetic architecture of Autoimmune Thyroid Disorder with clinical implications.</a> Am J Hum Genet, 82:1349-1356, 2008. (Featured Article). PMID: 18485327, PMC2427261, doi:10.1016/j.ajhg.2008.04.017</li>
                <li>Wratten NS, Memoli H, Huang Y, Dulencin AM, Matteson PG, Cornacchia MA, Azaro MA, Messenger J, Hayter JE, Bassett AS, Buyske S, Millonig JH, Vieland VJ, Brzustowicz LM. <a href="https://doi.org/10.1176/appi.ajp.2008.08081266">Identification of a schizophrenia associated functional non-coding variant in NOS1AP.</a> Amer J Psychiatry, 166:434-441, Mar 2, 2009. PMID: 19255043, PMC3295829, doi:10.1176/appi.ajp.2008.08081266</li>
                <li>Vieland VJ, Hallmayer J, Huang Y, Pagnamenta AT, Pinto D, Khan H, Monaco AP, Paterson AD, Scherer SW, Sutcliffe JS, Szatmari P; The Autism Genome Project (AGP). <a href="https://doi.org/10.1007/s11689-011-9072-9">Novel method for combined linkage and genome-wide association analysis finds evidence of distinct genetic architecture for two subtypes of autism.</a> J Neurodev Disord 3(2):113-123, 2011. PMID: 21484201, PMC3105232, doi:10.1007/s11689-011-9072-9</li>
                <li>Oikkonen J, Huang Y, Onkamo P, Ukkola-Vuoti L, Raijas P, Karma K, Vieland VJ, Järvelä I. <a href="https://doi.org/10.1038/mp.2014.8">A genome-wide linkage and association study of musical aptitude identifies loci containing genes related to inner ear development and neurocognitive functions.</a> Mol Psychiatry, March 11, 2014. PMID: 24614497, PMC4259854, doi:10.1038/mp.2014.8</li>
            </ul>
        </div>
        
        <input type="radio" name="tabs" id="projects">
        <label for="projects">What We Do</label>
        <div class="tab">
            
            <p>MathMed participates in a variety of biomedical research projects, primarily based upon application of our unique in-house approach to statistical genetics as implemented in the software package <span class="sc">Kelvin</span>. For a full list of grants and publications see <a href="">CV</a>. For <span class="sc">Kelvin</span> documentation, further information, and download, see <a href="https://kelvin.mathmed.org">kelvin.mathmed.org</a>.</p>
            
            <h2>Current Projects</h2>
            
            <h3>National Institute of Mental Health (NIMH) Repository & Genomic Resource (NRGR) (2012-present)</h3>
            
            <p>The NRGR has amassed a vast trove of genetic and phenotypic data collected over decades under NIMH-funded genetic protocols. Dr. Vieland leads a subproject within the NRGR called Combined Analysis of Psychiatric Disorders (CAPS). The primary goals of CAPS are enhancement of the NRGR data repository through extensive curation of archival clinical data from structured diagnostic instruments (e.g., the Diagnostic Interview for Genetic Studies, or DIGS), maintenance and management of newly deposited item-level clinical data, creation of customized tools for accessing and exploring fine-grained phenotypic information, and application of specialized analyses using <span class="sc">Kelvin</span> to illustrate and extend the utility of those data for genetic research purposes.</p>
            
            <p>The NRGR is administered via Cooperative Agreement (U24MH068457) between the NIMH and Rutgers University, with Drs. Linda Brzustowicz and Jay Tishfield as Principal Investigators. In addition to maintaining the data repository, the NRGR also maintains a transformed lymphocyte cell line repository of viable cells from individuals with psychiatric disorders and their relatives in frozen storage, and extracts and distributes DNA to NIMH-approved researchers. For further information on the NRGR and CAPS, visit <a href="https://www.nimhgenetics.org">nimhgenetics.org</a>.</p>
            
            <p>To read more about CAPS see:</p>
            <ul class="papers">
                <li>Walters KA, Huang Y, Azaro M, Tobin K, Lehner T, Brzustowicz LM, Vieland VJ. <a href="https://doi.org/10.1371/journal.pone.0084696">Meta-analysis of repository data: Impact of data regularization on NIMH schizophrenia linkage results.</a> PLoS One, Jan 14;9(1):e84696, 2014. doi:10.1371/journal.pone.0084696</li>
                <li>Vieland VJ, Walters KA, Lehner T, Azaro M, Tobin K, Huang Y, Brzustowicz LM. <a href="https://doi.org/10.1176/appi.ajp.2013.11121766">Revisiting schizophrenia linkage data in the NIMH repository: Reanalysis of regularized data across multiple studies.</a> Am J Psychiatry, 171(3):350-359, 2014. doi:10.1176/appi.ajp.2013.11121766</li>
                <li>Vieland VJ, Walters KA, Azaro M, Brzustowicz LM, Lehner T. <a href="https://doi.org/10.1159/000360003">The value of regenotyping older linkage data sets with denser marker panels.</a> Hum Hered 78(1):9-16, 2014. doi:10.1159/000360003</li>
            </ul>
            
            <h3>The Evidence Project (2006-present): Measuring the Evidence in Evidence-Based Medical Research</h3>
            
            <p>Most scientists take for granted that the true object of a data analysis in many (though not all) settings is to assess the strength of evidence for or against particular hypotheses. Most statisticians, however, focus on hypothesis testing and parameter estimation, which are quite different activities. A great deal of slippage in communication takes place when the results of these statistical procedures are interpreted by scientists in evidential terms. In a series of essays starting in 2006 Dr. Vieland began to rethink the problem of how best to measure the quantity of scientific interest – the evidence – from a measurement perspective, rather than in traditional statistical terms. This led to development of a novel body of theory involving the conservation and interconversion of different types of information with influx of new data during data analysis, and, more recently, the beginnings of a general approach to applying this theory in actual data analyses (although the applied methods are not yet ready for prime time).</p>
            
            <p>For further information on the Evidence Project, see:</p>
            <ul class="papers">
                <li>Vieland VJ. <a href="https://doi.org/10.1159/000093775">Thermometers: Something for statistical geneticists to think about.</a> Human Hered, 61:144-156, 2006. PMID: 16770079, doi:10.1159/000093775</li>
                <li>Vieland VJ. <a href="https://doi.org/10.1159/000324838">Where’s the evidence?</a> Hum Hered, 71(1):59-66, 2011. doi:10.1159/000324838</li>
                <li>Vieland VJ, Das J, Hodge SE, Seok S-C. <a href="https://doi.org/10.1007/s12064-013-0180-9">Measurement of statistical evidence on an absolute scale following thermodynamic principles.</a> Theory in Biosciences 132(3):181-194, 2013. PMID: 23463577, PMC3742421, doi:10.1007/s12064-013-0180-9</li>
                <li>Vieland VJ. <a href="https://doi.org/10.1159/000367599">Evidence, temperature and the laws of thermodynamics.</a> Hum Hered 78(3):153-163, 2014. doi:10.1159/000367599</li>
                <li>Vieland VJ, Seok S-C. <a href="https://doi.org/10.3390/e17085333">Statistical evidence measured on a properly calibrated scale across nested and non-nested hypothesis comparison.</a> Entropy 17:5333-5352, 2015. doi:10.3390/e17085333</li>
                <li>Vieland VJ, Seok S-C. <a href="https://doi.org/10.3390/e18040114">Statistical evidence measured on a properly calibrated scale for multinomial hypothesis comparisons.</a> Entropy 18(4):114, 2016. doi:10.3390/e18040114</li>
                <li>Vieland VJ. <a href="https://doi.org/10.1086/693931">Measurement of Statistical Evidence: Picking up Where Hacking (et al.) Left Off.</a> Philosophy of Science 84(5):853-865, 2017. doi:10.1086/693931</li>
                <li>Vieland VJ, Chang H. <a href="https://doi.org/10.1007/s11229-017-1666-7">No Evidence Amalgamation without Evidence Measurement.</a> Synthese 196:3139-3161, 2019. DOI: 10.1007/s11229-017-1666-7, 2018.</li>
                <li>Vieland VJ. <a href="https://doi.org/10.48550/arXiv.1805.11516">Absolutely 0 Evidence.</a> arXiv:1805.11516, 2018. doi:10.48550/arXiv.1805.11516</li>
            </ul>
            
            <h3>Genetic Modifiers of Duchenne Muscular Dystrophy (DMD) (2014-present)</h3>
            
            <p>This study involves a <span class="sc">Kelvin</span>-based genome-wide search for genetic modifiers of DMD disease progression and severity, with the goals of improving the precision of clinical treatment and providing better stratification of patients into clinical trials. DMD is a severe, progressive muscular dystrophy that affects 1 per 5,000 live male births. This project has been funded by a grant from the National Institutes of Health (R01NS085238), and we are currently anticipating a five-year renewal of the grant to begin July 2022, with Multiple Principal Investigators (MPIs) Drs. Kevin Flanigan, Veronica Vieland and Bob Weiss.</p>
            
            <p>To read about some of our current findings see:</p>
            <ul class="papers">
                <li>Weiss RB, Vieland VJ, Dunn DM, Kaminoh Y, Flanigan KM. United Dystrophinopathy Project. <a href="https://doi.org/10.1002/ana.25283">Long-range genomic regulators of THBS1 and LTBP4 modify disease severity in Duchenne muscular dystrophy.</a> Ann Neurol 84(2):234-245, 2018. doi:10.1002/ana.25283</li>
                <li>Vieland VJ, Seok S-C, Stewart WCL. <a href="https://doi.org/10.1371/journal.pone.0232300">A new linear regression-like residual for survival analysis, with application to genome wide association studies of time-to-event data.</a> PLoS One, 15(5):e0232300, 2020. doi:10.1371/journal.pone.0232300</li>
                <li>Vieland VJ, Seok S-C. <a href="https://doi.org/10.1371/journal.pone.0257164">The PPLD has advantages over conventional regression methods in application to moderately sized genome-wide association studies.</a> PLoS One. 2021 Sep 22;16(9):e0257164, 2021. doi:10.1371/journal.pone.0257164</li>
                <li>Flanigan KM, Waldrop MA, Martin PT, Alles R, Dunn DM, Alfano LN, Simmons TR, Moore-Clingenpeel M, Burian J, Seok S-C, Vieland VJ, Weiss RB. <a href="https://doi.org/10.1101/2021.11.03.21265899">Candidate gene modifiers of dystrophinopathy identified by the uniform application of genome-wide datasets to novel GWAS-identified loci.</a> medRxiv 2021.11.03.21265899; doi:10.1101/2021.11.03.21265899</li>
                <li>Flanigan KM, Waldrop MA, Martin PT, Alles R, Dunn DM, Alfano LN, Simmons TR, Moore-Clingenpeel M, Burian J, Seok S-C, Weiss RB, Vieland VJ. <a href="https://doi.org/10.1101/2021.11.03.21265887">A genome-wide association analysis of loss of ambulation in dystrophinopathy patients suggests multiple candidate modifiers of disease severity.</a> medRxiv 2021.11.03.21265887; doi:10.1101/2021.11.03.21265887</li>
            </ul>
            
            <h3>PediAtric ReseArch of Drugs, Immunoparalysis and Genetics during MODS (PARADIGM) (2019-2025)</h3>
            
            <p>The ultimate goal of PARADIGM is to reduce the incidence of immunoparalysis in pediatric multiple organ dysfunction syndrome (MODS) and its associated morbidities, by determining diagnosis-specific, treatment-specific, and host-specific (genetic) factors. The project is funded by National Institutes of Health grant R01HD095976, Drs. Mark Hall and Athena Zuppa, co-Principal Investigators. Dr. Vieland is responsible for using Kelvin to investigate the role of genetic factors in MODS-association immunoparalysis.</p>
            
            <h3>Overlapping Genetics of Major Depression and Pain (2020-2022)</h3>
            
            <p>Dr. Vieland serves as advisor to Genetics Fellow Dr. Daniel Nolan, MD, PhD (Nationwide Children’s Hospital). Dr. Nolan’s project is utilizing item-level diagnostic interview data along with genetic data from the NRGR to investigate possible overlap of genetic factors between depression and pain.</p>
            
            <h3>Estimation of Penetrance for Rare Variants (2022)</h3>
            
            <p>In collaboration with Dr. Andrew Paterson (SickKids Hospital, Toronto), Drs. Vieland and Seok are investigating the effects of different schemes for ascertaining patients in the course of clinical genome sequencing on estimates of the penetrance, or the probability that an individual carrying a rare, potentially pathogenic, variant will develop variant-related disease. This issue is important for genetic counseling, since over- or under- estimating the penetrance can lead to inaccurate genetic risk predictions; it is also important in ascribing and understanding pathogenicity for any given variant.</p>
        </div>
        
        <input type="radio" name="tabs" id="software">
        <label for="software" class="lasttab">Our Software</label>
        <div class="tab">
            <p>We’ll have downloads for all our software available shortly.</p>
        </div>
    </div>
    <p id="copyright">&copy; 2022 Mathematical Medicine LLC. <a href="https://creativecommons.org/licenses/by-nc/4.0/">CC BY-NC</a> except where otherwise noted.<br>General: <a href="mailto:contactus@mathmed.org">contactus@mathmed.org</a>  Website: <a href="mailto:web@mathmed.org">web@mathmed.org</a></p>
</body></html>
